test/proto/perf_tests/perf_control.proto

```diff
@@ -124,13 +75,15 @@ message LoadParams {
     UniformParams uniform = 2;
     DeterministicParams determ = 3;
     ParetoParams pareto = 4;
+    // No need to separately specify Closed-Loop as that
```

Comment:
We should still define ClosedLoopParams close_loop even when they are empty. We might need to add some params for close_loop in the future.

Also, looks like having all the behaviors part of oneOf enum will be easier to comprehend.